### PR TITLE
cleanup(OWNERS): remove inactive approvers

### PR DESCRIPTION
**Pull request by jasondellaluce**
**What type of PR is this?**

/kind cleanup

/kind documentation

**Any specific area of the project related to this PR?**

**What this PR does / why we need it**:

For https://github.com/falcosecurity/falco/issues/2115, this PR moves inactive approvers (who had very little or zero contributions over the past 6 months) from `approvers` to `emeritus_approvers` in OWNERS.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

File: falco/OWNERS [hunk 1]
```diff
@@ -1,6 +1,6 @@
-approvers:
-- bencer
-- nestorsalceda
 reviewers:
 - bencer
 - nestorsalceda
+emeritus_approvers:
+- bencer
+- nestorsalceda
```
Comment (cpanato) [inline falco/OWNERS:6]: nit: new line

**Comment by cpanato** [comment]
and will need to bump the chart version, then i recommend to split in two prs for each chart

**Comment by cpanato** [comment]
ops, sorry pressed the wrong button

**Review by fntlnz — CHANGES_REQUESTED** [review]
Please restore `fntlnz` https://github.com/falcosecurity/falco/issues/2115#issuecomment-1176045086

**Comment by jasondellaluce** [comment]
> and will need to bump the chart version, then i recommend to split in two prs for each chart

@cpanato I think you have a good point here, specially considering that the Falco chart has been recently released and deeply refactored. I'm gonna close this PR and split the OWNERS review in two.

**Pull request closed without merging**